test/test_constellations.py

```diff
@@ -0,0 +1,12 @@
+from constellations import ConstellationsWidget
```

Comment:
This test needs some refactoring to be compatible with Pytest, but we can worry about this in a later PR :)